app/bundles/PageBundle/Views/Trackable/click_counts.html.php

```diff
@@ -10,14 +10,29 @@
             </tr>
             </thead>
             <tbody>
-            <?php foreach ($trackables as $link): ?>
+                <?php 
+                    $totalClicks = 0;
+                    $totalUniqueClicks = 0;
+                    foreach ($trackables as $link):
+                            $totalClicks +=  $link['hits'];
+                            $totalUniqueClicks +=  $link['unique_hits'];
+                    endforeach;
+                ?>
                 <tr>
-                    <td class="long-text"><a href="<?php echo $link['url']; ?>"><?php echo $link['url']; ?></a></td>
-                    <td class="text-center"><?php echo $link['hits']; ?></td>
-                    <td class="text-center"><?php echo $link['unique_hits']; ?></td>
-                    <td><?php echo $link['redirect_id']; ?></td>
+                    <td class="long-text">Total Clicks</td>
+                    <td class="text-center"><?php echo $totalClicks; ?></td>
+                    <td class="text-center"><?php echo $totalUniqueClicks; ?></td>
+                    <td></td>
                 </tr>
-            <?php endforeach; ?>
+                
+                <?php foreach ($trackables as $link): ?>
+                    <tr>
+                        <td class="long-text"><a href="<?php echo $link['url']; ?>"><?php echo $link['url']; ?></a></td>
+                        <td class="text-center"><?php echo $link['hits']; ?></td>
+                        <td class="text-center"><?php echo $link['unique_hits']; ?></td>
+                        <td><?php echo $link['redirect_id']; ?></td>
+                    </tr>
+                <?php endforeach; ?>
```

Comment:
You can avoid iterating the `$trackables` twice by putting the total clicks row at the end of the table, and move the counting logic from lines 14-20 into the foreach beginning on line 28.

Also, "Total Clicks" should be translated. You can replace it with `$view['translator']->trans('mautic.core.total_clicks');`, then add that translation string into the `app/Bundles/CoreBundle/Translations/en_US/messages.ini` with a value of "Total Clicks"